source/miscellaneous.rst

```diff
@@ -58,14 +58,46 @@ Go to ``Bharat-sim Master\\src\\test\\scala\\com.bharatsim.engine\\graph\\neo4j`
 Using args in main method
 ==========================
 While using the ``run`` command on the sbt shell, one can pass in some ``string`` arguments. These arguements can be called by the main function. 
-for example:
+For example, one might want to change the name of the output file everytime they run the code. Instead of changing the output name manually each time, one can write a code like the one given below.
+
 .. code::
 
     var outputName = "dummyName"
 
-    def main(args: Array[String]): Unit = {
+      def main(args: Array[String]): Unit = {
         outputName = args(0)
-         
-The above block of code is allocating a variable called \verb|outputName|, and then using the 1st string passed through the  ``run`` command. For example, ``run "example"``. In this case, ``outputName`` becomes ``"example"``. The variable can be used by the user where ever they need it. 
+        var beforeCount = 0
```

Comment:
I didn't mean add so much complicated code! I just meant close the curly braces! I would suggest removing everything from line 69 to 99. I would also suggest changing lines 65 to 68 as follows: start off with the main declaration on line 67. Next, say ``val outputName = args(0)``. No need to mention the ``dummyName`` etc which is a choice that _we_ made. This way, you can explain that if you _don't_ give in an argument, the code will throw an error. (Please also show a screenshot of the error that occurs if you do this.)